setup.py

```diff
@@ -1,7 +1,15 @@
 import os
 import sys
+import re
 from setuptools import setup, find_packages
-from sys import platform
+
+# Exit if python version is unsupported.
+MINIMUM_PYTHON_VERSION = 3, 6, 0
+MAXIMUM_PYTHON_VERSION = 3, 7, 9
```

Comment:
I set the micro versions on the upper bound because if MAXIMUM_PYTHON_VERSION = 3, 7 then any micro version above 3.7.0 becomes unsupported, e.g., 3.7.1 will cause an exit. The lower bound micro version is there for consistency.

Will set to 3.8 next time I'm at a computer

Reply:
Hm I would think supporting 3.7 means supporting all versions of 3.7. So I think setting the maximum to 3.8.0 makes more sense.

Reply:
See 524e39b
Will do

Reply:
be522eb